packages/@lwc/template-compiler/src/codegen/static-element-serializer.ts

```diff
@@ -6,7 +6,8 @@
  */
 import { htmlEscape, HTML_NAMESPACE, isVoidElement } from '@lwc/shared';
 import { ChildNode, Comment, Element, Literal, Text } from '../shared/types';
-import { isElement, isText, isComment } from '../shared/ast';
+import { isElement, isText, isComment, isExpression } from '../shared/ast';
+import CodeGen from './codegen';
```

Comment:
```suggestion
import type CodeGen from './codegen';
```
If you're only using it for the type, then `import type` tells TypeScript that it can remove the import when compiling.

Reply:
I ended up removing the type but thanks for the tip! TIL